src/components/CampaignSummary.tsx

```diff
@@ -0,0 +1,170 @@
+    return `${moment.utc(date).format("MMMM Do YYYY")} (UTC)`;
+  }, []);
+
+  return (
+    <InfoCard
+      className={classes.card}
+      title={
+        <span className={classes.title}>
+          {campaignByService.status === "passing" && (
+            <CheckCircleIcon
+              className={classes.success}
+              titleAccess="Passing Campaign Icon"
+            />
+          )}
+          {campaignByService.status === "failing" && (
+            <CancelIcon
+              className={classes.failure}
+              titleAccess="Failing Campaign Icon"
+            />
+          )}
+          <span>{campaignByService.campaign?.name ?? "Campaign"}</span>
+        </span>
+      }
+    >
+      <Grid container>
+        <Grid item xs={12} lg={campaignByService.items?.nodes ? 6 : 12}>
+          <Typography>
+            <TeamOutlined className={classes.ownerIcon} />
+            <Link
+              href={`${opslevelUrl}${campaignByService.campaign?.owner?.href}`}
+            >
+              {campaignByService.campaign?.owner?.name ?? "Unknown"}
+            </Link>
+          </Typography>
+        </Grid>
+        {campaignByService.items?.nodes && (
+          <Grid
+            className={classes.nodeStatusesWrapper}
+            item
+            xs={12}
+            lg={6}
+            container
+            alignItems="center"
+            justifyContent="flex-start"
+          >
+            <span className={classes.nodeStatusWrapper}>
+              <CancelIcon
+                className={classes.failure}
+                titleAccess="Failing Checks Icon"
+              />
+              {
+                campaignByService.items.nodes.filter(
+                  (node) => node.status === "failed",
+                ).length
+              }
+            </span>
+            <span className={classes.nodeStatusWrapper}>
+              <CheckCircleIcon
+                className={classes.success}
+                titleAccess="Passing Checks Icon"
+              />
+              {
+                campaignByService.items.nodes.filter(
+                  (node) => node.status === "passed",
+                ).length
+              }
+            </span>
+          </Grid>
+        )}
+      </Grid>
+      {campaignByService.campaign?.status && (
+        <Typography>
+          Status:&nbsp;
+          <ColoredCampaignStatus status={campaignByService.campaign.status} />
+        </Typography>
+      )}
+
+      {campaignByService.campaign?.startDate && (
+        <Typography>
+          Start Date: {formatDate(campaignByService.campaign.startDate)}
+        </Typography>
+      )}
+      {campaignByService.campaign?.targetDate && (
+        <Typography>
+          Target Date: {formatDate(campaignByService.campaign.targetDate)}
+        </Typography>
+      )}
+      <div />
+      <div className={classes.actions}>
+        <Button variant="text">
+          <Link href={`${opslevelUrl}${campaignByService.campaign?.href}`}>
```

Comment:
Can this this open a new window instead?

Reply:
Also related to links -- there's a link to the whole campaign, can there be a link to the campaigns tab on this service instead/additionally?

Reply:
> Can this this open a new window instead?


Updated!

Reply:
> can there be a link to the campaigns tab on this service instead/additionally

Hmmm, I'm not opposed but don't think this is worth blocking this release on. Mind making a ticket for it if you think it's worthwhile?